### PR TITLE
i18n(zh-cn): Update `basics/astro-pages.mdx` and `reference/api-reference.mdx`

### PR DESCRIPTION
**Pull request by 100gle**
#### Description (required)

Update `basics/astro-pages.mdx` and `reference/api-reference.mdx`

Upstream PR: #8573

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 1]
```diff
@@ -1692,6 +1692,36 @@ import { Code } from 'astro:components';
 
 该组件在构建时为代码块提供语法高亮（不包括客户端 JavaScript）。该组件由 Shiki 驱动，它支持所有流行的[主题](https://shiki.style/themes)和[语言](https://shiki.style/languages)。另外，你可以通过给 `theme` 和 `lang` 传递自定义主题和语言分别添加它们。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/api-reference.mdx:1693]: 这里还缺少了东西

File: src/content/docs/zh-cn/basics/astro-pages.mdx [hunk 2]
```diff
@@ -110,6 +110,25 @@ title: '我的 Markdown 页面'
 
 在开发过程中，如果你有一个 `500.astro` 文件，运行时抛出的错误将被记录在终端中，而不是显示在错误覆盖层中。
 
+### `error`
+
+<p><Since v="4.11.0" /></p>
+
+`src/pages/500.astro` 是一个特殊页面，它会在渲染过程中对抛出的任何错误都会自动传递一个 `error` 属性。这允许你使用错误细节（例如来自页面、中间件等）来向访客展示信息。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/basics/astro-pages.mdx:117]:
```suggestion
`src/pages/500.astro` 是一个特殊页面，它会在渲染过程中对抛出的任何错误都会自动传递一个 `error` 属性。这允许你向访客展示详细的错误信息（例如来自页面、中间件等）。
```
这样翻译是不是会好点，没有直译，但意思没变

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 3]
```diff
@@ -1692,6 +1692,36 @@ import { Code } from 'astro:components';
...
 
+#### 转换器
+
+
+<p><Since v="4.11.0" /></p>
+
+[Shiki 转换器](https://shiki.tmrs.site/packages/transformers#shikijs-transformers) 能够被额外地被应用到代码上，即通过将它们作为数组传递给 `transformers` 属性。
+
+注意，`transformers` 只能用于类，同时你必须提供自己的 CSS 规则来定位代码块的元素。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/api-reference.mdx:1702]:
```suggestion
注意，`transformers` 只能用于类，同时你必须提供自己的 CSS 规则来针对代码块的元素。
```

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 4]
```diff
@@ -1692,6 +1692,36 @@ import { Code } from 'astro:components';
 
 该组件在构建时为代码块提供语法高亮（不包括客户端 JavaScript）。该组件由 Shiki 驱动，它支持所有流行的[主题](https://shiki.style/themes)和[语言](https://shiki.style/languages)。另外，你可以通过给 `theme` 和 `lang` 传递自定义主题和语言分别添加它们。
 
+#### 转换器
+
+
+<p><Since v="4.11.0" /></p>
+
+[Shiki 转换器](https://shiki.tmrs.site/packages/transformers#shikijs-transformers) 能够被额外地被应用到代码上，即通过将它们作为数组传递给 `transformers` 属性。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/api-reference.mdx:1700]:
```suggestion
[Shiki 转换器](https://shiki.tmrs.site/packages/transformers#shikijs-transformers) 可以通过将其作为数组传递到 `transformers` 属性中，来选择性地应用于代码。
```

**Review by liruifengv — APPROVED** [review]
LGTM. Thank you!

**Review by dreyfus92 — APPROVED** [review]
Thank you peeps! LGTM ✅

**Pull request merged**